### ISSUE TITLE
Disabling vanilla items

### ISSUE THREAD

**Issue by Pelirow** [AnonymousProductions/MineFantasy2#19]
I'm not sure how Iguana Tinker Tweaks did it, but it makes it so vanilla tools and weapons have a durability of 1 so they break on the first use. 

Would be a great idea to apply that mechanic to the iron/diamond tools, swords, and armor. That way they're available for other mods that need them for crafting, but forces players to use MF stuff in gameplay. Unless they want to use gold items/armor, which is not sustainable so I wouldn't suggest you configure gold stuff. Wood and Stone tools can be left alone since they're needed to get to copper.

**Comment by AnonymousProductions** [comment]
1 use seems silly, but I believe it to be possible

**Comment by Pelirow** [comment]
I don't think you can make a "zero use" tool, so Iguana's Tinker Tweaks made the items have just 1 point of durability. Tiny bit silly, but keeps mod compatibility.

**Comment by AnonymousProductions** [comment]
Nah, id just make them maybe half as durable, you CAN use them... but its a waste
